platform/android/export/export.cpp

```diff
@@ -744,8 +744,40 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return OK;
 	}
 
-	void _fix_manifest(const Ref<EditorExportPreset> &p_preset, Vector<uint8_t> &p_manifest, bool p_give_internet) {
+	void _get_permissions(const Ref<EditorExportPreset> &p_preset, bool p_give_internet, Vector<String> *perms) {
```

Comment:
```suggestion
	void _get_permissions(const Ref<EditorExportPreset> &p_preset, bool p_give_internet, Vector<String> *r_perms) {
```

Reply:
Should the `p_give_internet` stuff be refactored to use `r_perms` like the rest?

Reply:
`p_give_internet` might be true even if it is not checked off by the user. [Here](https://github.com/godotengine/godot/pull/39998/files/7b8332ef3294710cd1655782343fdc9db39981ec#diff-fb88394f2df1faf844deaff3b3639aeeR2563) is where the `p_give_internet` variable originates